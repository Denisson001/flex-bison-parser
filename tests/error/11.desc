Compilation error - invalid number expression