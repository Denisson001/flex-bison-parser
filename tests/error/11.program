{
    c = 3;
    a = b + 5 / c + c ** 10;
}
