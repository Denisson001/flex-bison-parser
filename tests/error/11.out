Parser: —Åompilation error - syntax error, unexpected '*', expecting _NUMBER or _NUMBER_VAR or _LEN_FUNC or '(' near line 3
